apps/freeablo/faworld/player.cpp

```diff
@@ -54,7 +54,26 @@ namespace FAWorld
     void Player::initCommon()
     {
         mWorld.registerPlayer(this);
-        mInventory.equipChanged.connect([this]() { updateSprites(); });
+        mInventory.inventoryChanged.connect([this](BasicInventory const& inventory, Item const& removed, Item const& added) {
+            (void)inventory;
+            (void)removed;
+
+            // Update player graphics.
+            updateSprites();
+
+            if (!added.isEmpty())
+            {
+                // Play inventory place sound.
+                std::string soundPath = added.getInvPlaceSoundPath();
+                Engine::ThreadManager::get()->playSound(soundPath);
```

Comment:
Two checks need to be added here:
1. We shouldn't play these when it's the initial items being added during character creation (maybe add a "silent" flag to inv transfers somehow).
2. We should only play these when the player is the local player (== FAWorld::World::getCurrentPlayer)